roadmap/2022.md

```diff
@@ -0,0 +1,55 @@
+# Roadmap 2022
+
+This is the roadmap planned by the Integration team at MeiliSearch for the year 2022.
+
+Nothing is definitive: the roadmap can be adapted depending on the need of the users and the company.
+
+## 2022: the Main Lines
+
+#### Objectives
+
+- Better know our users and our community to ease our decision making
+  - set the analytics on every integration
+  - Extract information from them to prioritize our tasks
+- Keep working with the community, improve our way of working with the contributors by creating clearer issues, and by providing more and more "good first issues".
+- Work with the @meilisearch/devrel-team to have help for the support for answering issues. The goal is to allow the integration team developers to focus on the PR reviews and their coding tasks, rather than doing support.
+- Work with the @meilisearch/docs-team to provide complete and easy-to-maintain documentation for each SDK
+- Be more organized regarding our rhythm of working
+    - during the pre-release period: improve the process and define steps for each sprint
+    - during the implementation period: find the right balance between maintainance and new content addition.
+- Release our first database integrations since we don't provide any in 2021.
```

Comment:
```suggestion
- Road to v1! In 2022, MeiliSearch is starting the implementation and the stabilization of the API regarding the release of the first stable version of MeiliSearch (v1.0.0). The integration team must start to think about it and adapt their integrations in this v1 objective. For example, the stabilization of the most popular integrations will be necessary.
- Release our first database integrations since we don't provide any in 2021. 
```